conf/gulp-tasks/library.gulpfile.js

```diff
@@ -28,18 +29,41 @@ exports.dev = function devJSBundle () {
 };
 
 /**
- * Creates a build task per locale, and combines them into a series task.
+ * Creates a build task for each provided locale and combines them into a series task.
+ * @param {Array<string>} locales
  * @returns {Promise<Function>}
  */
-exports.default = function defaultJSBundle () {
-  const localizedTaskPromises = BUILD_LOCALES.map(locale => {
+function createJSBundlesForLocales (locales) {
```

Comment:
Are we creating JS bundles per locale here or per language? I was thinking we would only "generate" or "build" assets per language and make copies of that per-locale. If that is the case, we should be careful in our naming here so as not to cause confusion.

Reply:
The bundles created are simply per language. I used "Locales" in the name to indicate that this function is capable of building bundles for locales. But I see why it can be unclear, so I'll rename it!